src/content/docs/en/reference/dev-toolbar-app-reference.mdx

```diff
@@ -197,104 +197,104 @@ This event can be sent from your plugin to change the state of your plugin. This
 
-### `astro-dev-overlay:PLUGIN_ID:initialized`
+### `astro-dev-toolbar:APP_ID:initialized`
 
-This message is sent when the plugin is initialized. The data for this message is empty.
+This message is sent when the app is initialized. The data for this message is empty.
 
 ```ts title="integration.ts"
 {
   // ...
   "astro:server:setup": ({ server }) => {
-    server.ws.on("astro-dev-overlay:super-plugin:initialized", () => {
-      console.log("My plugin was initialized!");
+    server.ws.on("astro-dev-toolbar:super-app:initialized", () => {
+      console.log("My app was initialized!");
     });
   },
   // ...
 }
 ```
 
-### `astro-dev-overlay:PLUGIN_ID:toggled`
+### `astro-dev-toolbar:APP_ID:toggled`
 
-This message is sent when the user clicks on the plugin icon in the UI. The data for this message is a boolean indicating whether the plugin is enabled or not.
+This message is sent when the user clicks on the app icon in the UI. The data for this message is a boolean indicating whether the app is enabled or not.
 
 ```ts title="integration.ts"
 {
   // ...
   "astro:server:setup": ({ server }) => {
-    server.ws.on("astro-dev-overlay:super-plugin:toggled", (data) => {
-      console.log(`My plugin is now ${data.state ? "enabled" : "disabled"}!`);
+    server.ws.on("astro-dev-toolbar:super-app:toggled", (data) => {
+      console.log(`My app is now ${data.state ? "enabled" : "disabled"}!`);
     });
   },
   // ...
 }
 ```
 
 :::note
-The built-in `connection` event from Vite fires **before** dev overlay plugins are initialized and therefore cannot be used directly by plugins. Instead, use the `astro-dev-overlay:PLUGIN_ID:initialized` event.
+The built-in `connection` event from Vite fires **before** Dev Toolbar apps are initialized and therefore cannot be used directly by apps. Instead, use the `astro-dev-toolbar:APP_ID:initialized` event.
 :::
 
 ## UI Toolkit
 
-The dev overlay includes a set of web components that can be used to build plugins with a consistent look and feel.
+The Dev Toolbar includes a set of web components that can be used to build apps with a consistent look and feel.
 
-### `astro-dev-overlay-window`
+### `astro-dev-toolbar-window`
 
 Shows a window with optionally a title and an icon.
 
-`window-title` is a string that will be shown at the top of the overlay window. `window-icon` can either be a string of a SVG file or an icon from [the icon list](#icons).
+`window-title` is a string that will be shown at the top of the toolbar window. `window-icon` can either be a string of a SVG file or an icon from [the icon list](#icons).
 
 The slot of the component will be used as the content of the window.
 
 ```html
-<astro-dev-overlay-window window-title="MyWindows" window-icon="astro:logo">
+<astro-dev-toolbar-window window-title="MyWindows" window-icon="astro:logo">
   <p>My content</p>
-</astro-dev-overlay-window>
+</astro-dev-toolbar-window>
 ```
 
-### `astro-dev-overlay-card`
+### `astro-dev-toolbar-card`
```

Comment:
``````suggestion
When building a window using JavaScript, slotted content must go inside the light DOM of the component.

```js
const myWindow = document.createElement('astro-dev-toolbar-window');
const myContent = document.createElement('p');
myContent.textContent = 'My content';

// use appendChild directly on the window
myWindow.appendChild(myContent);
```

### `astro-dev-toolbar-button`

Shows a button.

The slot of the component will be used as the content of the button.

```js
const myButton = document.createElement('astro-dev-toolbar-button');
myButton.textContent = 'Click me!';
myButton.buttonStyle = "purple";
myButton.size = "medium";

myButton.addEventListener('click', () => {
  console.log('Clicked!');
});
```

#### `size`

The size of the button (small, medium, large).

#### `button-style`

The style of the button (`ghost`, `outline`, `purple`, `gray`, `red`). When using `ghost`, the button itself is invisible and only the content of the button will be shown.

In JavaScript, set this property using the `buttonStyle` property to avoid conflict with the native `style` property.

### `astro-dev-toolbar-badge`

Shows a badge.

The slot of the component will be used as the content of the badge.

```html
<astro-dev-toolbar-badge>My badge</astro-dev-toolbar-badge>
```

#### `size`

The size of the badge (`small`, `large`).

#### `badge-style`

The style (color) of the badge (`purple`, `gray`, `red`, `green`, `yellow`).

In JavaScript, set this property using the `buttonStyle` property to avoid conflict with the native `style` property.

### `astro-dev-toolbar-card`
``````
This was all new content in #5561 - I believe I've updated for the renamings here, but please check!

Reply:
Why was this heading removed, was it intentional? ``### `astro-dev-toolbar-card` ``